docs/en/observability/profiling-self-managed-ops.asciidoc

```diff
@@ -0,0 +1,228 @@
+Allocate data to hot nodes for best performance and user experience.
+If storage becomes a concern, tune the data retention by customizing the Universal Profiling <<profiling-ilm-custom-policy, index lifecycle management policy>>.
+
+[discrete]
+[[profiling-self-managed-ops-configuration]]
+== Configure the collector and symbolizer
+
+You can configure the collector and symbolizer using the YAML file and CLI flags, with the CLI flags taking precedence over the YAML file.
+The configuration files are created during the installation process, as seen in <<profiling-self-managed-running-linux-configfile, Create configuration files section>>.
+Comments in the configuration files explain the purpose of each configuration option.
+
+Restart the backend binaries after modifying the configuration files for changes to take effect.
+
+[discrete]
+[[profiling-self-managed-ops-configuration-cli-overrides]]
+=== Use CLI flags to override configuration file values
+
+When building configuration options for each of the backend binaries, you can use CLI flags to override the values in the YAML configuration file.
+The overrides **must** contain the full path to the configuration option and must be in a key=value format. For example, `-E application.field.key=value`, where `application` is the name of the binary.
+
+For example, to enable TLS in the HTTP server of the collector, you can pass the `-E pf-elastic-collector.ssl.enabled=true` flag.
+This will override the `ssl.enabled` option found in the YAML configuration file.
+
+[discrete]
+[[profiling-self-managed-ops-monitoring]]
+== Monitoring
+
+Monitor the collector and symbolizer through <<profiling-self-managed-ops-monitoring-logs>> and <<profiling-self-managed-ops-monitoring-metrics>> to ensure the services are running and healthy.
+Without both services running, profiling data will not be ingested and symbolized,
+and querying Kibana won't return data.
+
+[discrete]
+[[profiling-self-managed-ops-monitoring-logs]]
+=== Logs
+
+The collector and symbolizer always log to standard output.
+You can turn on debug logs by setting the `verbose` configuration option to `true` in the YAML configuration file.
+
+Avoid using debug logs in production, as they can be very verbose and impact backend performance.
+Only enable debug logs when troubleshooting a failed deployment or when instructed to do so by support.
+
+Logs are formatted as "key=value" pairs, and {es} and {kib} can automatically parse them into fields.
+
+ A log collector, such as Filebeat, can collect and send logs to {es} for indexing and analysis.
+Depending on how it's installed, a Filebeat input of type `journald` (for OS packages), `log` (for binaries), or `container` can be used to process the logs.
+Refer to the https://www.elastic.co/guide/en/beats/filebeat/current/configuring-howto-filebeat.html[filebeat documentation] for more information.
```

Comment:
@mdbirnstiehl all of these cross-book links need to be updated to use our [versioned attributes](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc).